bundles/org.openhab.binding.mail/NOTICE

```diff
@@ -0,0 +1,21 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0/.
+
+== Source Code
+
+https://github.com/openhab/openhab2-addons
+
+
+Apache Commons Mail
```

Comment:
Please use the same notation as in  other NOTICE files:
```
commons-email:
* License: Apache 2.0 License
* Project; https://commons.apache.org/proper/commons-email
* Source:  https://commons.apache.org/proper/commons-email
```

Maybe also add something about `javax.mail`?